connectors/airwatch/src/main/service/rhel/files/default-conf/managed-apps.yml

```diff
@@ -1,16 +1,37 @@
-# Details about managed apps. Platform, app name and app id.
+# Details about managed apps.
 # You must override these details in /etc. By default there are no managed apps.
-android:
-
-ios:
+airwatch:
+  apps:
+    -
```

Comment:
Currently, we are configuring apps manually in managed-apps.yml. This method will become cumbersome if the no of apps are more. We should provide an option in the console to configure apps and keywords and expose an API to save it in ```managed-apps.yml```. We can take this feature as an enhancement to ```airwatch-connector```.

Reply:
When no of managed apps increase, it will become cumbersome to supply details of all of them. 
I'm also wondering how many managed app might be there on an average. And also how frequently customers change managed app list in Airwatch console.